Cargo.toml

```diff
@@ -9,6 +9,7 @@ documentation = "https://docs.rs/slog-json"
 homepage = "https://github.com/slog-rs/slog"
 repository = "https://github.com/slog-rs/json"
 readme = "README.md"
+edition = "2018"
```

Comment:
Do you think we can get away without it? Unless there's a really good reason,  would rather not have these couple of folks stuck with older compiler or something, have to upgrade.

Reply:
Absolutely. That's why I split the PR into cf7d07a7f036bd6a82fe816c904e051cd7d6f438 (which adds the new `serialize` function) and e0a0f69206027e6493f77056bf2ea517dfb43b15 (which upgrades to 2018 without changing any functionality).